hosting/.vscode/settings.json

```
@@ -2,4 +2,8 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
   },
+  "editor.tabSize": 2,
+  "files.eol": "\n",
+  "files.trimFinalNewlines": true,
+  "files.insertFinalNewline": true,
 }
```

```suggestion
}

```

末尾改行いるよねーとは思うけどあんまり理由はしらない
UNIXのなんかだっけ？

---

https://qiita.com/Aseiide/items/e66be44c4b7972902063
> POSIX上、テキストファイルは「行」の集合体で、そして「行」は「0文字以上の文字+改行」からなっている、とのことです（Qiita）。
へぇ～